docs/For User/Account keys/02 Multisig.md

```diff
@@ -0,0 +1,256 @@
+  "signatures": []
+}
+```
+
+### Step 3: Sign individually
+
+Sign with `test1` and `test2` and create individual signatures.
+
+```linux
+imversed tx sign \
+    unsignedTx.json \
+    --multisig=imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    --from=test1 \
+    --output-document=test1sig.json \
+    --chain-id=imversed_9000-4
+```
+```linux
+imversed tx sign \
+    unsignedTx.json \
+    --multisig=imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    --from=test2 \
+    --output-document=test2sig.json \
+    --chain-id=imversed_9000-4
+```
+
+### Step 4: Create multisignature
+Combine signatures to sign transaction.
+
+```linux
+```
+
+The TX is now signed:
+
+```linux
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.bank.v1beta1.MsgSend",
+        "from_address": "imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft",
+        "to_address": "imversed157g6rn6t6k5rl0dl57zha2wx72t633axqyvvwq",
+        "amount": [
+          {
+            "denom": "aimv",
+            "amount": "5000000000000000000"
+          }
+        ]
+      }
+    ],
+    "memo": "",
+    "timeout_height": "0",
+    "extension_options": [],
+    "non_critical_extension_options": []
+  },
+  "auth_info": {
+    "signer_infos": [
+      {
+        "public_key": {
+          "@type": "/cosmos.crypto.multisig.LegacyAminoPubKey",
+          "threshold": 2,
+          "public_keys": [
+            {
+              "@type": "/cosmos.crypto.secp256k1.PubKey",
+              "key": "ApCzSG8k7Tr4aM6e4OJRExN7cNtvH21L9azbh+uRrvt4"
+            },
+            {
+              "@type": "/cosmos.crypto.secp256k1.PubKey",
+              "key": "Ah91erz8ChNanqLe9ea948rvAiXMCRlR5Ka7EE/c0xUK"
+            },
+            {
+              "@type": "/cosmos.crypto.secp256k1.PubKey",
+              "key": "A0OjtIUCFJM3AobJ9HJTWKP9RZV2+WPcwVjLgsAidrZ/"
+            }
+          ]
+        },
+        "mode_info": {
+          "multi": {
+            "bitarray": {
+              "extra_bits_stored": 3,
+              "elems": "wA=="
+            },
+            "mode_infos": [
+              {
+                "single": {
+                  "mode": "SIGN_MODE_LEGACY_AMINO_JSON"
+                }
+              },
+              {
+                "single": {
+                  "mode": "SIGN_MODE_LEGACY_AMINO_JSON"
+                }
+              }
+            ]
+          }
+        },
+        "sequence": "1"
+      }
+    ],
+    "fee": {
+      "amount": [
+        {
+          "denom": "aimv",
+          "amount": "1000000"
+        }
+      ],
+      "gas_limit": "200000",
+      "payer": "",
+      "granter": ""
+    }
+  },
+  "signatures": [
+    "CkCEeIbeGc+I1ipZuhp/0KhVNnWAv2tTlvgo5x61lzk1KHmLPV38m/YFurrFt5cm5+fqIXrn+FlOjrJuzBhw8ogYCkCawm9mpXsBHk0CFsE5618fVnvScEkfrzW0c2jCcjqV8EPuj3ut74UWzZyQkwtJGxUWtro9EgnGsB7Di1Gzizst"
+  ]
+}
+```
+
+### Step 5: Broadcast transaction
+
+```linux
+imversed tx broadcast signedTx.json \
+    --chain-id=imversed_9000-4 \

```

Comment:
Change **imversed_9000-4** to **imversed_5555555-1**

Reply:
Updated